lib/paypal_nvp.rb

```diff
@@ -31,6 +31,17 @@ def initialize(sandbox = false, extras = {})
     end
     @extras = extras
     @rootCA = @rootCA || '/etc/ssl/certs'
+
+
+    if config
+      # Get the timeout options from the config
+      @open_timeout = config[type]["open_timeout"]
+      @read_timeout = config[type]["read_timeout"]
+    else
+      # Get the timeout options from the extra parameter
+      @open_timeout = extras[:open_timeout]
+      @read_timeout = extras[:read_timeout]
+    end
```

Comment:
@jaxi Would it not be better to include these in the existing `if config` conditional above starting on line 19?